src/component/Options.js

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import { buildDeck, selectCards, shuffleCards } from "../functions/Deck";
+import Table from "./Table";
+
+const GAME_CARDS = 21
+var deck = buildDeck();
```

Comment:
Não vejo motivo pra não usarmos `const` aqui também.
```suggestion
const deck = buildDeck();
```